### ISSUE TITLE
Included randomizer settings appear to be corrupted

### ISSUE THREAD

**Issue by snoozbuster** [besteon/Ironmon-Tracker#345]
The first three characters of all the provided rqms files appear to be corrupted. They should be `320` but are other non-printables (eg `@h` in the FRLG ones). This means they don't load correctly. I got the correct settings string from [this gist](https://gist.github.com/UTDZac/a147c497424dfbd537d8c4b0c22b5621/41d92d64b428afa2d93f15cf3c7e466cedf0c6ca#fire-red-leaf-green) (note that this is an older revision, as the latest revision uses 4.6.0 strings) and confirmed that it's only the first three characters that appear corrupted.

**Comment by snoozbuster** [comment]
okay, never mind - I guess the randomizer program adds some other binary data at the beginning of the settings string. Either way, I couldn't get the included files to load, I had to find the settings strings and save a new copy directly from the randomizer program (which looks identical to the ones I got from the repo - I would attach it to this comment, but GH doesn't allow that).

**Comment by UTDZac** [comment]
Thank you for the report about the randomizer settings files included with the tracker. A few questions:

Q1. What is the exct version of the Ironmon Tracker and the exact version of the Randomizer(s) you tested with?

Q2. Which of the below settings files were you testing with?

![image](https://user-images.githubusercontent.com/4258818/218292732-72d1e288-c5c5-4b1c-9e83-85e0e1904097.png)

For context, I manage both the settings files found within the tracker and the ones found on the gist you provided. Prior to the latest Ironmon Tracker release, I verify the included settings files would load properly with either Randomizer 4.5.1 or Randomizer 4.6.0. While the latter does output an error message in the `RandomizerErrorLog.txt` file found in your Tracker folder, its merely a warning and doesn't block the auto generate a rom feature.

**Comment by snoozbuster** [comment]
I was using 4.5.1 for the randomizer and the latest ironmon tracker. the file I was specifically trying was the FRLG Kaizo settings file

**Comment by UTDZac** [comment]
I'm not sure how to see the files you see or your setup. But the Tracker settings files that come included do appear to work properly for my Randomizer(s) and my tracker. This is the first report i've heard about this, so maybe its an issue on your end with your computer's environment?

Are any of your files in a `zip` file? If they are still "compressed" then chances are they will not work properly.

**Comment by snoozbuster** [comment]
I loaded up the files in HxD just to see what was up. in case there was like newlines at the end that were messing it up or something. top is the one that came with this repo, bottom is the one that works for me. highlighted is the section that's different between the two.

![image](https://user-images.githubusercontent.com/5778364/218297834-89839313-5fab-489d-a25a-f8085f8e4dda.png)

the `DHUFnG` variant is the one I get if I load `320WRIEEjIBAAQABwCRAAKeBnsECQEAFAACCQAuEgAAAAAABRi45ATkAYYICTIGBAIyAAUAEEZpcmUgUmVkIChVKSAxLjGwV2lQ48M4ig==` (the one in the linked gist) in 4.5.1's UI and then save it as a rnqs file. I'm quite sure it's 4.5.1 as it says so in the randomizer UI. 

All this being said, I had to redownload a fresh version of the rnqs file from the repo for this comparison and I figured I should try it again to see if I was still getting the error... but now it just works. I have no idea why it didn't before, I didn't change anything about the randomizer or the other settings. before I would just keep getting an error about the randomizer being unable to read the settings file. Since neither of us can repro anymore I'll close the issue - if I can get a consistent repro for you I'll come back with more details.
